.gitignore

```diff
@@ -2,5 +2,5 @@
 .idea
 target
 dev.yaml
-.mvn
 .DS_Store
+.mvn
```

Comment:
We already chatted about this, but this should not be gitignored, I think this directory needs to be pushed upstream!

Reply:
Fixed in https://github.com/GoogleCloudPlatform/anthos-edge-usecases/pull/2/commits/9e1f980cda2340019e2e2ef67ad52523e0641335 by merging the previous branch